sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -36,6 +36,18 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
   /**
    * For each node, extract properties in the form of a list ['key1', 'key2', 'key3', 'value']
    * into a pair (key1.key2.key3, value).
+   *
```

Comment:
Not very related to this PR. @hvanhovell Do you know when we will have something like
```
TOK_TABLEPROPERTY
  'key_part1'
  'key_part2'
  'key_part3'
  'value'
```
(the form that matches the original comment `extract properties in the form of a list ['key1', 'key2', 'key3', 'value'] into a pair (key1.key2.key3, value).`)